sentences/en/_common.yaml

```diff
@@ -96,13 +96,14 @@ lists:
 expansion_rules:
   name: "[the] {name}"
   area: "[the] {area}"
-  what_is: "(what's | whats | what is)"
-  brightness: "{brightness}[%| percent]"
-  turn: "(turn | switch | change)"
-  temp: "(temp | temperature)"
-  temperature: "{temperature}[°| degrees] [{temperature_unit}]"
+  what_is: "(what's|whats|what is)"
+  brightness: "{brightness}[%|percent]"
```

Comment:
this would not match `20 percent` (i.e. with a space), but only `20percent`. I'd change it to `"{brightness}[ ][%|percent]"`, maybe?

Reply:
I did add the optional space now, but as the tests did still pass, it doesn't seem required though. I'll leave it in though, to make clear both options will work.